src/wind_energy/BoussinesqBubble.cpp

```diff
@@ -0,0 +1,71 @@
+#include "BoussinesqBubble.H"
+#include "BoussinesqBubbleFieldInit.H"
+#include "incflo.H"
+
+#include "AMReX_ParmParse.H"
+#include "AMReX_MultiFab.H"
+
+namespace amr_wind {
+
+BoussinesqBubble::BoussinesqBubble(incflo* incflo_in)
+    : m_incflo(incflo_in)
+{
+    amrex::ParmParse pp("bb");
+
+    pp.query("use_boussinesq", m_has_boussinesq);
+
+    // Instantiate the BoussinesqBubble field initializer
+    m_field_init.reset(new BoussinesqBubbleFieldInit());
+
+    if (m_has_boussinesq)
+        m_boussinesq.reset(new BoussinesqBuoyancy());
```

Comment:
We should probably have `BoussinesqBuoyancy` take a `std::string` argument that allows use to pass `BoussinesqBuoyancy("abl")` in `ABL.cpp` and `BoussinesqBuoyancy("bb")` here and use the appropriate namespace to lookup parameters.